docs/content/testing/testing-overview.md

```diff
@@ -0,0 +1,43 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing overview
+
+We use testing to validate functionality and to ensure that the parts of the Unity GDK and the framework as a whole is resilient under different conditions.
```

Comment:
(1) We tend to use 'we' in documentation to refer to Improbable (unlike code comments where it refers to what 'we' the coder(s) are doing with the code). Just confirming that "we" means Improbable here. I assume yes. :-)

(2) "...are resilient under..." (plural subject -  "parts"  as well as the framework).

Reply:
Assume these tests are for anyone adding to the Unity GDK itself. Not for anyone using the GDK to make a SpatialOS project in Unity? If so, we need to make that clear that this is what these tests are for, I think?

Reply:
"We use tests to validate..." is fine.